specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
+    mysql = require('mysql'),
+    parse = require('csv-parse'),
+    expect = require('chai').expect,
+    DB_CONFIG = require('../dbconfig.json'),
+    sinon = require('sinon');
+
+describe('When the user has csv files in the files folder', function () {
+    this.timeout(10000);
+
+    beforeEach(function () {
+        return ensureTablesRemoved(['t1', 't2', 'tr']);
+    });
+
+    afterEach(function () {
+        return ensureTablesRemoved(['t1', 't2', 'tr']);
+    });
+
+    describe('And runs the app.js from the command line with no options', function () {
+        beforeEach(() => {
+            return runCommand('node app.js');
+        });
+
+        it('Should complete without error', function () {
+        });
+
+        it('Should have inserted the t1 csv as expected', function (done) {
+            var connection = mysql.createConnection(DB_CONFIG);
```

Comment:
We are using this code very often... i.e. create connection, run a query, get the results, etc... We should turn it into a function which returns a promise. It would make our tests much cleaner.